external-builds/pytorch/build_prod_wheels.py

```diff
@@ -314,6 +314,13 @@ def add_env_compiler_flags(env: dict[str, str], flagname: str, *compiler_flags:
     print(f"-- Appended {flagname}+={append}")
 
 
+def find_dir_containing(file_name: str, *possible_paths: Path) -> Path:
```

Comment:
Is there a style preference for `*possible_paths: Path` over `possible_paths: Iterable[Path]` with `from typing import Iterable`, or `list[Path]`?

Down below, I'd more naturally prefer:
```python
    triton_python_dir = find_dir_containing(
        "setup.py", [triton_dir / "python", triton_dir]
    )
```

Over what is currently there:

```python
    triton_python_dir = find_dir_containing(
        "setup.py", triton_dir / "python", triton_dir
    )
```